backend/tracim_backend/applications/content_file/controller.py

```diff
@@ -421,7 +432,9 @@ def preview_pdf_revision(self, context, request: TracimRequest, hapic_data=None)
         content = api.get_one(hapic_data.path.content_id, content_type=ContentTypeSlug.ANY.value)
         if content.file_extension in app_config.PREVIEW__SKIPLIST:
```

Comment:
Same remark here - I'm not sure you already have access to the file extension but if you have, you may skip the get_one()

Reply:
ok